.github/workflows/prt_labels.yml

```diff
@@ -0,0 +1,45 @@
+name: Remove the PRT label, for the new commit
+
+on:
+  pull_request:
+    types: ["synchronize"]
+
+jobs:
+    prt_labels_remover:
+      name: remove the PRT label when amendments or new commits added to PR
+      runs-on: ubuntu-latest
+      if: "(contains(github.event.pull_request.labels.*.name, 'PRT-Passed') || contains(github.event.pull_request.labels.*.name, 'PRT-Failed'))"
+      steps:
+        - name: Fetch the PRT status
+          id: prt
+          uses: omkarkhatavkar/wait-for-status-checks@main
+          with:
+            ref: ${{ github.head_ref }}
+            context: 'Robottelo-Runner'
+            wait-interval: 2
+            count: 5
+
+        - name: remove the PRT Passed/Failed label, for new commit
+          if: always() && ${{steps.prt.outputs.result}} == 'not_found'
```

Comment:
Is this check really needed to verify if the PRT outputs result is not found ? 

I think we can keep it simple, On PR refresh just remove the labels regardless of PRT status.

Reply:
But then, this will never work in case when: 
1. PRT is run has a check for PRT in PR checks
2. Added a new commit, that would run this workflow of label removals

And this is the most common case in robottelo. So question, Does PRT check is always removed and ASAP new commit/ammendments is pushed to PR on all PRs ?

Reply:
> 1. PRT is run has a check for PRT in PR checks
First of all this check not to see if PRT is passed or Failed. It is checking the that PRT is `Not Found` (which literally means there is new commit as PRT result goes away with status API). This is pretty quick check that will run after new commit. If someone does the PRT comment even in that case it will done. This is seperate GHA. So it will never disturb the main PRT cherry-picks or auto-merge GHA's      

> 2. Added a new commit, that would run this workflow of label removals
Isn't that we want. We wanted to remove the labels related to `PRT-Passed/Failed`  in case new commit as the code has been updated. Previous PRT result not valid now. 

> . So question, Does PRT check is always removed and ASAP new commit/ammendments is pushed to PR on all PRs ?
That's true. Hence I added the check in PRT GHA that get's me the `Not Found` Result. Leverage that result to remove the `PRT-Passed/Failed` labels. 

I know it's bit tricky to understand. I will have more direct in person conversation with you so that it become bit easier to grab some internal details.

Reply:
offline discussion, adding some more sleep would really help in for avoiding the race condition